testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergHiveViewsCompatibility.java

```diff
@@ -75,7 +76,6 @@ public void testIcebergHiveViewsCompatibility()
                             .add(row("hive_view_qualified_hive"))
```

Comment:
@losipiuk 

```
.add(row("iceberg_table")) // TODO: should this be filtered out?
```

filtering out the entry from show tables would involve querying the properties metadata of each table.

Can I remove this TODO ?